geometry/dbox.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "geometry/bbox.hpp"
+#include "geometry/point2d.hpp"
+
+namespace m2
+{
+class DBox
```

Comment:
Please provide a short description of this class.

Reply:
Done